pages/about.tsx

```diff
@@ -277,75 +334,81 @@ const AboutPage = () => {
         </p>
 
         <p className="pb-8">
-          The <code>memory_byte_size</code> can be obtained with{' '}
-          <RelativeLink to="#59" title="MSIZE" />. We can see that the cost
-          grows quadratically with the size, making higher offsets more costly
-          and discouraging to use too much memory. Any opcode accessing memory
-          may trigger an expansion (including, for example,{' '}
-          <RelativeLink to="#51" title="MLOAD" />,{' '}
+          The <code>memory_byte_size</code> can be obtained with opcode{' '}
+          <RelativeLink to="#59" title="MSIZE" />. The cost of memory expansiont
+          triggered by <RelativeLink to="#59" title="MSIZE" /> grows
+          quadratically, disincentivizing the overuse of memory by making higher
+          offsets more costly. Any opcode that accesses memory may trigger an
+          expansion (such as <RelativeLink to="#51" title="MLOAD" />,{' '}
           <RelativeLink to="#F3" title="RETURN" /> or{' '}
           <RelativeLink to="#37" title="CALLDATACOPY" />
-          ). Each opcode that can is mentioned in the{' '}
-          <RelativeLink title="reference" />. Note also that an opcode with a
-          byte size parameter of 0 will not trigger a memory expansion,
-          regardless of its offset parameters.
+          ). Use our <RelativeLink title="reference" /> to review which opcode
+          is capable of accessing memory. Note that opcodes with a byte size
+          parameter of 0 will not trigger memory expansion, regardless of their
+          offset parameters.
         </p>
       </SectionWrapper>
 
-      <SectionWrapper header={<H3>Access sets</H3>} anchorKey="accesssets">
+      <SectionWrapper header={<H3>Access Sets</H3>} anchorKey="accesssets">
+        <p className="pb-6">
+          Access sets are defined per transaction, and not per call. Access sets
+          can be thought of as two independent types of lists: those of touch
+          contract addresses, and those of touched contract storage slots.
+        </p>
+        <p className="pb-6">
+          When a contract address is initialized by a transaction, instruction,
```

Comment:
A contract address is not initialised, would be better to say "when an address is accessed"
Also, should not say _contract_ address, because all addresses are put in the set, even an EoA.

Reply:
done